<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>批改实验</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #f5f5f5;
    font-family: &quot;Microsoft YaHei&quot;, Arial;
}
QLabel {
    color: #333333;
    font-size: 14px;
    font-weight: bold;
    padding: 5px 0;
}
QLineEdit {
    padding: 8px;
    border: 1px solid #dddddd;
    border-radius: 4px;
    background-color: white;
    font-size: 13px;
}
QTextEdit {
    padding: 8px;
    border: 1px solid #dddddd;
    border-radius: 4px;
    background-color: white;
    font-size: 13px;
}
QPushButton {
    padding: 8px 20px;
    font-size: 13px;
    border: none;
    border-radius: 4px;
    color: white;
}
QPushButton#pushButton {
    background-color: #757575;
}
QPushButton#pushButton:hover {
    background-color: #616161;
}
QPushButton#pushButton_2 {
    background-color: #4CAF50;
}
QPushButton#pushButton_2:hover {
    background-color: #45a049;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <property name="spacing">
    <number>15</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <widget class="QLabel" name="title_label">
       <property name="styleSheet">
        <string notr="true">font-size: 18px;
padding: 10px 0;</string>
       </property>
       <property name="text">
        <string>批改实验报告</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>8</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>实验主题</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>8</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>实验内容</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>200</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>8</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>评语</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>请输入评语...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>得分</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>输入分数</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>返回</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>提交</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
